sonar-slang-plugin/src/main/java/com/sonarsource/slang/kotlin/KotlinSensor.java

```diff
@@ -54,7 +55,9 @@
 
   public KotlinSensor(CheckFactory checkFactory, FileLinesContextFactory fileLinesContextFactory, NoSonarFilter noSonarFilter) {
     checks = checkFactory.create(SlangPlugin.KOTLIN_REPOSITORY_KEY);
-    checks.addAnnotatedChecks((Iterable<?>) CommonCheckList.checks());
+    // TODO: Add logic for rules that require language specific configuration at construction time
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.slang%3Aslang&pullRequest=127&issues=AWRbS0H6-1T-AjkxlrYG&open=AWRbS0H6-1T-AjkxlrYG)

[ISSUE_KEY]: AWRbS0H6-1T-AjkxlrYG